torch/csrc/jit/codegen/cuda/lower_unroll.cpp

```diff
@@ -124,40 +124,43 @@ kir::Expr* findVectorizedSet(
 
 // Get full extent for the inner-most, merged root domain
 kir::Val* getVectorizeExtent(
-    kir::TensorView* tv,
-    const std::vector<kir::Val*>& indices) {
-  auto domain = tv->domain()->hasRFactor() ? tv->domain()->rfactorDomain()
-                                           : tv->domain()->rootDomain();
+    kir::TensorView* global_tv,
+    kir::TensorView* local_tv) {
+  TORCH_INTERNAL_ASSERT(!global_tv->domain()->hasRFactor());
```

Comment:
Should we check that `global_tv` has valid contiguity information?
Options:
1. `global_tv` is in global memory
2. `global_tv` is either an input or output to the fusion

Reply:
Can contiguity information be invalid?